tests/core/test_initialize.py

```diff
@@ -1,14 +1,12 @@
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
-
-
 from distutils.sysconfig import get_python_lib
 from logging import getLogger
 import ntpath
 import os
 from os.path import abspath, dirname, isfile, join, realpath
 import sys
-from unittest import TestCase
+from unittest.mock import patch
```

Comment:
there is a pytest fixture called `mocker`

Reply:
It is the unittest.mock interface but it cleans up automatically. Example https://github.com/conda/conda-package-handling/blob/conda-package-handling-2.0/tests/test_cli.py#L30